src/pages/MapViewer/components/Toolbox/components/ToolboxRow/index.tsx

```diff
@@ -4,10 +4,10 @@ import { ToolboxItemProps } from './types.js';
 
 import './styles.scss';
 
-const ToolboxRow = ({ thumbnail, title, dataPoints, onClick }: ToolboxItemProps) => {
+const ToolboxRow = ({ thumbnail, title, dataPoints, onClick, children }: ToolboxItemProps) => {
   return (
-    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
-    <div className="toolbox-row" onMouseUp={onClick}>
+    // eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events
```

Comment:
Should we not be changing these as we go, or is it intended to be its own separate ticket?  As I see a lot of eslint disabling going on throughout and cautious it's going to build up.

Reply:
To fix this is means a bit refactoring, it means replacing any event handlers on divs,span etc with a button. The button will most likely need to be styled so it doesn't look like a button though, so there is enough going on that in my opinion, there should be a ticket just for this.

From now on though, we should not be adding event handlers to static elements, so it shouldn't build up from what we currently have.